pages/docs/tutorials/validate-documents.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Studio AsyncAPI document validation"
+description: This tutorial will teach us how to validate AsyncAPI documents using the AsyncAPI Studio tool.
+weight: 120
+---
+
+# Introduction
+This tutorial will teach you how to validate AsyncAPI documents using the [AsyncAPI Studio tool](https://studio.asyncapi.com/).
+
+We will start with a broken AsyncAPI document and troubleshoot via console errors step-by-step until we end up with a valid AsyncAPI document. This process will illustrate how to identify [`REQUIRED` properties in AsyncAPI documents](https://www.asyncapi.com/docs/reference/specification/v2.5.0#A2SObject).
+
```

Comment:
@starlightknown Hey, you're missing the `Installation Guide` section. Please add it and tag us to let us know when it's ready to review again.

Reply:
Hey we discussed about this in slack and you told me that we don't need installation guide. This is done in studio so we don't need anything to install locally. Let me know your thoughts

Reply:
Oooh you're right, we talked about this 😂. Sorry, my bad 🤪